### ISSUE TITLE
Windows Support

### ISSUE THREAD

**Issue by thiboot** [driftyco/ionic-plugin-deploy#110]
Hello,
Amazing feature !
Could you consider carrying this on Windows 10 platform ?

**Comment by ericb** [comment]
It's definitely been discussed, but this is not in active development. I'll keep this ticket around for future reference.

**Comment by thiboot** [comment]
Ok, thanks :-)

**Comment by snehasis** [comment]
Any update on this Windows Support , as its been a long time now , did the development started now ?

**Comment by digaus** [comment]
Yes that would be awesome... Since windows is officially supported by Ionic one should also be able to update the app just like on iOS and Android
